src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs

```diff
@@ -212,7 +212,10 @@ private static bool IsSameLocalOrField(BoundExpression expr1, BoundExpression ex
 
         private static bool IsComCallWithRefOmitted(MethodSymbol method, ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> argumentRefKindsOpt)
         {
-            if ((object)method.ContainingType == null || !method.ContainingType.IsComImport) return false;
+            if (method.ParameterCount != arguments.Length ||
```

Comment:
Should we simply check for HasErrors in VisitCall instead? That should take care of all kinds of error conditions we might run into now or in the future.